src/sc2renderer/sc2_renderer.cc

```diff
@@ -135,8 +135,13 @@ void Matrix8BPPPlayers(const char* bytes, int w_mat, int h_mat, int off_x, int o
 void ImageRGB(const char* bytes, int width, int height, int off_x, int off_y) {
     assert(renderer_);
     assert(window_);
-
+#if defined(__linux__)
+    SDL_Surface* surface = SDL_CreateRGBSurfaceWithFormatFrom((void *) bytes, width, height, 24, 3 * width,
```

Comment:
`(void *) bytes` -> `(void*)bytes`

Reply:
Ok, from what I've found this is a shorter form (i.e. without pixels) of the original SDL_CreateRGBSurface, so there is no need in defined guards, we should use it on all platforms or should correct the original pixels mask.